src/integration/queries/RunningSessionQuery.graphql

```diff
@@ -12,6 +12,7 @@ query RunningSession {
     feedbacks {
       id
       content
+      tags
```

Comment:
For the running session, we would definitely want to check in integration testing that the tags are already returned. But all of these changes only make sense if you also change/add tests for them :)

If you do not plan to extend the integration framework for these changes, please remove the changes in all files under src/integration. Otherwise, the integration test would quite probably break.